src/proxy.rs

```diff
@@ -703,7 +708,7 @@ impl fmt::Debug for ProxyScheme {
 }
 
 type SystemProxyMap = HashMap<String, ProxyScheme>;
-type RegistryProxyValues = (u32, String);
+type PlatformProxyValues = (u32, String);
```

Comment:
actually is there any reason we don't represent `PlatformProxyValues` as an `Option<String>` where `None` indicates there's no proxy set? AFAICT there's never a case we care about a proxy value if it's disabled. It would simplify a lot of the code since we could directly return `None` if there are no enabled proxies.

Reply:
I think originally, this was an easy tuple to represent the two variables coming out of the Windows registry. Agreed that we can use an `Option<String>` everywhere instead; made that change.